modules/installation-minimum-resource-requirements.adoc

```diff
@@ -245,20 +245,17 @@ ifdef::vsphere[]
 2. As with all user-provisioned installations, if you choose to use {op-system-base} compute machines in your cluster, you take responsibility for all operating system life cycle management and maintenance, including performing system updates, applying patches, and completing all other required tasks. Use of {op-system-base} 7 compute machines is deprecated and has been removed in {product-title} 4.10 and later.
 endif::vsphere[]
 --
-[NOTE]
-====
-As of {product-title} version 4.13, RHCOS is based on RHEL version 9.2, which updates the micro-architecture requirements. The following list contains the minimum instruction set architectures (ISA) that each architecture requires:
+
+ifdef::azure[]
+[IMPORTANT]
 
 * x86-64 architecture requires x86-64-v2 ISA
 * ARM64 architecture requires ARMv8.0-A ISA
 * IBM Power architecture requires Power 9 ISA
 * s390x architecture requires z14 ISA
 
-For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/9.0_release_notes/index#architectures[RHEL Architectures].
-====
+For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/9.2_release_notes/index#architectures[RHEL Architectures].
```

Comment:
The `main` branch points to 4.19 currently, is that right? It's based on RHEL 9.4
```suggestion
For more information, see link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html-single/9.4_release_notes/index#architectures[RHEL Architectures].
```